awx/ui/test/e2e/settings.js

```diff
@@ -10,6 +10,12 @@ const AWX_E2E_TIMEOUT_MEDIUM = process.env.AWX_E2E_TIMEOUT_MEDIUM || 5000;
 const AWX_E2E_TIMEOUT_SHORT = process.env.AWX_E2E_TIMEOUT_SHORT || 1000;
 const AWX_E2E_LAUNCH_URL = process.env.AWX_E2E_LAUNCH_URL || AWX_E2E_URL;
 
+// Screenshot capture settings
+const AWX_E2E_SCREENSHOTS_ENABLED = process.env.AWX_E2E_SCREENSHOTS_ENABLED || false;
+const AWX_E2E_SCREENSHOTS_ON_ERROR = process.env.AWX_E2E_SCREENSHOTS_ON_ERROR || true;
```

Comment:
This should probably default to false. Otherwise, the default behavior for existing systems / workflows might change.